### ISSUE TITLE
Unable to use package (vite related?)

### ISSUE THREAD

**Issue by mosheduminer** [amoutonbrady/solid-quill#1]
Using the TS template from solidjs/templates, I get the following error when I attempt to `import { SolidQuill } from "solid-quill"`.
```
Uncaught SyntaxError: The requested module '/node_modules/quill/dist/quill.js?v=f58f4410' does not provide an export named 'default'
```
Dependencies:
```
"devDependencies": {
    "vite": "^2.4.4",
    "vite-plugin-solid": "^2.0.1"
  },
  "dependencies": {
    "solid-app-router": "^0.1.2",
    "solid-js": "^1.0.2",
    "solid-quill": "^0.3.0"
  }
```

**Comment by amoutonbrady** [comment]
@mosheduminer It seems like it's a bug I can't put my finger on how to automatically include it in the `optimizeDeps` dependencies of vite.

I've spent the entire morning and things work locally or via a linked package but not when pulled from npm. As a result I've published 4 versions and none of them worked.

I take that the only solution for now is to manually include `quill` like so:

```
export default defineConfig({
  plugins: [solidPlugin()],
  optimizeDeps: {
    include: ["quill"],
  },
})
```

I'll leave this open in case anyone has an idea.
